### DISCUSSION TITLE
[월드 1.1.7] Multiplay 일시적인 룸 접속 제한하기

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#623] [5. Q&A]
월드 1.1.7 입니다 

Multiplay 일시적인 룸 접속 제한하기를 테스트 를 진행 중에 있는데요 
https://studio.zepeto.me/guides/temporarily-restricting-room-access

![image](https://user-images.githubusercontent.com/50188795/163540350-6266dca4-3664-493f-82d2-9ef00ef58054.png)

요부분을 다음과 같이 테스트 해봤습니다 
![image](https://user-images.githubusercontent.com/50188795/163540387-02cb46bd-bc8b-4e82-b927-21b906963952.png)

lock 부분은 출력 한 내용입니다 

제한 되면 room joined 가 안나와야 할거 같은데 출력 됩니다 
![image](https://user-images.githubusercontent.com/50188795/163540497-bb1e234c-a4fd-44f0-82f0-c7e5a962081d.png)

이부분 동작 하는것인가요?

**Comment by smilejsu82** [comment]
#48 확인 했습니다 

아직 수정 안되었나 보네요 ^^

**Comment by 17beans** [comment]
현재 clients.length가 undefined로 나오는 현상이 있습니다. clients.length 말고 this.state.players.size를 사용해 보시면 어떨까요?
참고로 말씀드리자면 현재 room unlock이 동작하지 않고 있습니다.

**Comment by smilejsu82** [comment]
> 현재 clients.length가 undefined로 나오는 현상이 있습니다. clients.length 말고 this.state.players.size를 사용해 보시면 어떨까요?
> 
> 참고로 말씀드리자면 현재 room unlock이 동작하지 않고 있습니다.

감사합니다 ^^ 참고할게요 ~ 
lock도 동작하지 않거나 최소 2명이거나 하는거 같아요 size에 문제가 아니라는것은 위 스크린샷에 "lock" 출력된것을 보면 알수 있습니다 😃 
참고로 size채크후 else 문에 넣어도 똑같이 room join 출력이 됩니다 😁 
또한 월드 셋팅에서는 최소 2명으로 밖에 설정이 안되는것을 미루어봐서 2명이하 lock은 동작 안할수도 있다고 생각하는 부분입니다
